server/src/test/java/org/opensearch/rest/action/admin/indices/RestValidateQueryActionTests.java

```diff
@@ -96,7 +96,7 @@ protected void doExecute(Task task, ActionRequest request, ActionListener listen
         final Map<ActionType, TransportAction> actions = new HashMap<>();
         actions.put(ValidateQueryAction.INSTANCE, transportAction);
 
-        client.initialize(actions, () -> "local", null, new NamedWriteableRegistry(Collections.emptyList()));
+        client.initialize(actions, null, () -> "local", null, new NamedWriteableRegistry(Collections.emptyList()));
```

Comment:
Can we create a dummy obj of `actionsRegistry` instead of passing `null`>

Reply:
Sure.